script/README.md

```diff
@@ -0,0 +1,148 @@
+# lua scripts
+
+[LUA](https://www.lua.org/)-scripts can be used as filters in skipper. The
+current implementation supports [Lua 5.1](https://www.lua.org/manual/5.1/).
+
+A script can be given as file (ending with `.lua`) or as inline code. The
+script needs to be the first parameter for the `lua` filter, e.g.
+* file `lua("/path/to/file.lua")`
+* inline `lua("function request(c, p); print(c.request.url); end")`
+
+Any other additional parameters for the filter must be `key=value` strings.
+These will be passed as table to the called functions as second parameter.
+**NOTE**: Any parameter starting with "lua-" should not be used to pass
+values for the script - those will be used for configuring the filter.
+
+A filter script needs at least one global function: `request` or `response`.
+If present, they are called with a skipper filter context and the params passed
+in the route as table like
+```lua
+-- route looks like
+--
+-- any: * -> lua("./test.lua", "myparam=foo", "other=bar") -> <shunt>
```

Comment:
What is the CWD in this case, used for ./test.lua ?
Can you add this to the docs please?